easybuild/easyconfigs/d/dftd3-lib/dftd3-lib-0.10-GCC-11.2.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'dftd3-lib'
+version = '0.10'
+
+homepage = 'https://github.com/dftbplus/dftd3-lib'
+description = """This is a repackaged version of the DFTD3 program by S. Grimme and his coworkers.
+The original program (V3.1 Rev 1) was downloaded at 2016-04-03. It has been
+converted to free format and encapsulated into modules."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'dftbplus'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+patches = ['dftd3-lib-%(version)s_fix-extras-syntax.patch']
+checksums = [
+    'db61bc6c7c699628e8c5bf2018ea38de03a53eac38014e06845829d765caf6bb',  # 0.10.tar.gz
+    'e22144773f045b5eadabad65aa93ad598faffda4a543f28def7ee90b4324f9aa',  # dftd3-lib-0.10_fix-extras-syntax.patch
+]
+
+parallel = 1
+
+buildopts = 'FC="$FC" FCFLAGS="$FCFLAGS" LNFLAGS="$LDFLAGS"'
+
+files_to_copy = [
+    (['prg/dftd3', 'test/testapi'], 'bin'),
+    (['lib/libdftd3.a'], 'lib'),
+    (['lib/*.mod', 'prg/*.mod'], 'include'),
+    (['doc/man.pdf', 'CHANGELOG.rst', 'LICENSE', 'README.rst'], 'share'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/dftd3', 'bin/testapi', 'lib/libdftd3.a'],
+    'dirs': ['include', 'share'],
+}
+
```

Comment:
Is there a sensible `sanity_check_command` that you could define? E.g. something like `<main_executable> --version` or `--help` is useful to check if the binary at least starts. That also ensures things like all shared libraries being found by the runtime linker.